e2e/keywords/common.resource

```diff
@@ -22,8 +23,10 @@ Set test environment
     Set Test Variable    ${deployment_list}
     @{statefulset_list} =    Create List
     Set Test Variable    ${statefulset_list}
+    setup_control_plane_network_latency
```

Comment:
Does this imply that network latency will affect all the tests?

Reply:
Yes, the default latency is `0`.

Reply:
Any reason we need this for all tests?

Reply:
A test case isn't restricted to only have one negative factor. For example, in this case we can have `node reboot` + `network latency`.

Reply:
Could the test cases run without this line?